### PR TITLE
tabs tracking

### PR DESCRIPTION
**Pull request by dendoesit**
## What?

Added tracking to Tabs in a new EventData

Can be checked here : https://web-onboardi-grw-577-ta-ec0eqz.herokuapp.com/se-en/new-member

## Why?

Ticket

image

### CODE REVIEW

File: src/client/pages/OfferNew/Perils/index.tsx [hunk 1]
```diff
@@ -43,9 +44,10 @@ export const Perils: React.FC<Props> = ({ offerData }) => {
     [offerData.quotes],
   )
 
-  const handleChangeTab = () => {
+  const handleChangeTab = (name: string) => {
     pushToGTMDataLayer({
-      event: 'offer-tab-click',
+      event: 'perils_tab_click',
+      eventData: { type: changeCase.snakeCase(name) },
```
Comment (gustaveen) [inline src/client/pages/OfferNew/Perils/index.tsx:50]: Did @sonnyandersson1 want the data in snack case as well?

**Review by gustaveen — COMMENTED** [review]
Looks good! Let's make a review app out of this so Sonny can try it out and validate the data 👍

File: src/client/pages/OfferNew/Perils/index.tsx (hunk 1 above)
Comment (dendoesit) [reply src/client/pages/OfferNew/Perils/index.tsx:50]: yeah. he specifically asked for that .

**Comment by dendoesit** [comment]
@gustaveen Deployed the review app. Got approval from Sonny.
Sonny Andersson  1:07 PM
Thanks a lot! Looks great :))

**Review by gustaveen — APPROVED** [review]
Great Dan! 👌

**Pull request merged**